src/app/journaling/JournalPage.tsx

```diff
@@ -0,0 +1,147 @@
+"use client";
+
+import MonthAndDayDate from "@/components/Date/Date";
+import { EditJournalEntry } from "@/components/JournalEntry/EditJournalEntry";
+import { ViewJournalEntry } from "@/components/JournalEntry/ViewJournalEntry";
+import { JSONContent } from "@tiptap/react";
+import { useState } from "react";
+
+const JournalPage = () => {
+  const [journalEntries, setJournalEntries] = useState<
+    {
+      content: JSONContent;
+      key: number;
+      date: Date;
+    }[]
+  >([
```

Comment:
Let's delete all of those. Keep the state empty, because we can add our own entries.